app/src/components/templateEditor/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const TemplateLanguages = {
+  FString: "f-string", // {variable}
+  Mustache: "mustache", // {{variable}}
```

Comment:
know this was pre-existing but could add to a doc comment with in an examples block so we can see this elsewhere - feel like you had that on some of the other stuff so maybe not necessary here